staging/src/k8s.io/controller-manager/pkg/clientbuilder/client_builder_dynamic.go

```diff
@@ -132,10 +132,10 @@ func (t *DynamicControllerClientBuilder) Config(saName string) (*restclient.Conf
 	return &configCopy, nil
 }
 
-func (t *DynamicControllerClientBuilder) ConfigOrDie(name string) *restclient.Config {
+func (t *DynamicControllerClientBuilder) ConfigOrDie(ctx context.Context, name string) *restclient.Config {
 	clientConfig, err := t.Config(name)
 	if err != nil {
-		klog.Fatal(err)
+		klog.FromContext(ctx).Error(err, "error in ConfigOrDie")
```

Comment:
This is a change in behavior: it no longer aborts.

https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#replacing-fatal-calls